Assets/Shapes2D/Scripts/Shape.cs

```diff
@@ -957,8 +958,8 @@ protected class ShaderProps {
                 var scaleFactor = image.canvas.scaleFactor;
                 if (image.canvas.renderMode == RenderMode.ScreenSpaceCamera)
                     scaleFactor = image.canvas.transform.lossyScale.x;
-                size.x = Vector3.Distance(topLeft, topRight) / scaleFactor;
-                size.y = Vector3.Distance(topLeft, bottomLeft) / scaleFactor;
+                size.x = Vector2.Distance(topLeft, topRight) / scaleFactor;
+                size.y = Vector2.Distance(topLeft, bottomLeft) / scaleFactor;
```

Comment:
Same as the above, I think if this isn't needed by the PR it might as well stay as Vector3 just in case it was solving some edge case